src/actions/test/getAutomine.ts

```diff
@@ -30,7 +30,13 @@ export type GetAutomineReturnType = boolean
 export async function getAutomine<TChain extends Chain | undefined>(
   client: TestClient<TestClientMode, Transport, TChain>,
 ): Promise<GetAutomineReturnType> {
-  return await client.request({
-    method: `${client.mode}_getAutomine`,
-  })
+  if (client.mode === 'ganache') {
+    return await client.request({
+      method: 'eth_mining',
+    })
+  } else {
+    return await client.request({
+      method: `${client.mode}_getAutomine`,
+    })
+  }
```

Comment:
Can short circuit instead of adding `else`

```suggestion
  if (client.mode === 'ganache')
    return await client.request({ method: 'eth_mining' })
  return await client.request({ method: `${client.mode}_getAutomine` })
```